.ansible-lint

```diff
@@ -0,0 +1,31 @@
+---
+skip_list:
+  # TODO: Fix and enable one-by-one over time
```

Comment:
But only the ones that make sense. `sanity[cannot-ignore]` looks pretty much wrong, for example.

Reply:
I hope that the eventual goal is to have both the collection(s) and ansible-lint rules in a state that no rule skips or any changes to default configs are necessary. For `sanity[cannot-ignore]` for example I am hardlocked on https://github.com/ansible/ansible/pull/79700 making it into ansible-core _and_ every version of ansible-core falling out of support until I can disable this in my collection. Similar with the shebangs in this repo here apparently?

There are some easy wins with fixing the long tail of errors (end of line whitespace and such) and automatic rewrites could be used or introduced for some things too. There are however also some issues with that (e.g. https://github.com/ansible/ansible-lint/issues/3226 will hit every collection created from the default template).